app/main/appium.js

```diff
@@ -321,9 +321,9 @@ function connectClientMethodListener () {
 }
 
 const getCurrentSessions = _.debounce(async (evt, data) => {
-  const {host, port, ssl} = data;
+  const {host, port, path, ssl} = data;
```

Comment:
this is overriding the default import, which is also named `path`. Would it be possible to rename the prop or just use `data.path` format to avoid shadowing?

Reply:
I would do something like....

`const {host, port, path:appiumPath, ssl} = data;`

Reply:
oops... nice..